libs/shared/ui/src/lib/components/table-primitives/table-primitives.tsx

```diff
@@ -0,0 +1,121 @@
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+interface TableRootProps extends ComponentPropsWithoutRef<'table'> {}
+
+const TableRoot = forwardRef<ElementRef<'table'>, TableRootProps>(function TableRoot(
+  { children, className, ...rest },
+  ref
+) {
+  return (
+    <table
+      ref={ref}
+      className={twMerge('table-fixed border-collapse rounded text-sm text-neutral-400', className)}
+      {...rest}
+    >
+      {children}
+    </table>
+  )
+})
+
+interface TableHeaderProps extends ComponentPropsWithoutRef<'thead'> {}
+
+const TableHeader = forwardRef<ElementRef<'thead'>, TableHeaderProps>(function TableHeader({ children, ...rest }, ref) {
+  return (
+    <thead ref={ref} {...rest}>
+      {children}
+    </thead>
+  )
+})
+
+interface TableBodyProps extends ComponentPropsWithoutRef<'tbody'> {}
+
+const TableBody = forwardRef<ElementRef<'tbody'>, TableBodyProps>(function TableBody({ children, ...rest }, ref) {
+  return (
+    <tbody ref={ref} {...rest}>
+      {children}
+    </tbody>
+  )
+})
+
+interface TableRowProps extends ComponentPropsWithoutRef<'tr'> {}
+
+const TableRow = forwardRef<ElementRef<'tr'>, TableRowProps>(function TableRow({ children, className, ...rest }, ref) {
+  return (
+    <tr ref={ref} className={twMerge('hover:bg-neutral-100', className)} {...rest}>
```

Comment:
I think by default, we don't add `hover:bg-neutral-100`, because in some case we don't need it (ex: row from `thead`)